### DISCUSSION TITLE
Store Hacs sparito con aggiornamento HA

### DISCUSSION THREAD

**Discussion by michele-diurno** [indomus/forum#3180] [HUB: Home Assistant]
Ciao a tutti,
ho eseguito l'aggiornamento di HA 2020.08 e come per magia è scomparso HACS e Alexa (Haska) non funziona più.
Qualcuno può spiegarmi cosa sia successo e come si può recuperare?
Grazie.

**Comment by snakuzzo** [comment]
Di sicuro nessuno può risponderti basandosi su quello che hai scritto.
Premesso che non ci sono problemi su nessuno dei due con l'aggiornamento...
Per HACS, svuota la cache del browser e controlla.
Per haaska non so che dirti.

In ogni caso, se dovessero esserci problemi, ne avresti evidenza nei log all"avvio di core

**Comment by michele-diurno** [comment]
Ho svuotato la cache e riavviato HA e tutto è tornato a funzionare.
Grazie mille per il consiglio, forse sono stato frettoloso nello scrivere il post.
